pwa/src/App.tsx

```diff
@@ -28,6 +29,15 @@ const App: React.FunctionComponent = () => {
                         <PatientInfoForm />
                     </Wrapper>
                 </Route>
+                <Route path="/diagnosis">
+                    <Wrapper
+                        navTitle="Diagnosis"
+                        leftArrowText="Patient Info"
+                        leftArrowRoute="/patientinfo"
```

Comment:
This routing is temporary, right? As in, everything should be on one page once all the components are individually reviewed.

Reply:
Per the feedback from Prof. Johnson, yes. 
I think we should discuss that change with the entire team, and then that should potentially be it's own issue and PR once the individual components are completed.

Reply:
Created #92